ProjFS.Mac/PrjFSLib/PrjFSUser.cpp

```diff
@@ -33,49 +71,77 @@ io_connect_t PrjFSService_ConnectToDriver(enum PrjFSServiceUserClientType client
     }
     
     // Check kernel's interface version matches ours or we're asking for trouble
```

Comment:
nit: This comment seems unnecessary, the method name 'ValidateVersion' seems like enough info for me.

Reply:
Good point, thanks; I've removed the comment… and also added an underscore in the new function's name to more closely match the convention.